prow/config.yaml

```diff
@@ -121,9 +121,9 @@ presubmits:
             path: /mnt/disks/ssd0
 
   - name: pull-kubernetes-cross
-    context: pull-kubernetes-cross
-    rerun_command: "@k8s-bot pull-kubernetes-cross test this"
-    trigger: "@k8s-bot pull-kubernetes-cross test this"
+    context: Jenkins cross build
+    rerun_command: "@k8s-bot cross test this"
```

Comment:
I don't mind changing it, but for consistency, here's what the bazel build trigger is: https://github.com/kubernetes/test-infra/blob/master/prow/config.yaml#L45-L50

Is it still a good idea to change it?